actors/common/src/test/java/org/sunbird/feed/impl/FeedServiceImplTest.java

```diff
@@ -0,0 +1,140 @@
+import org.sunbird.models.user.Feed;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+  ServiceFactory.class,
+  ElasticSearchRestHighImpl.class,
+  ElasticSearchHelper.class,
+  EsClientFactory.class,
+  CassandraOperationImpl.class,
+  ElasticSearchService.class,
+  FeedServiceImpl.class,
+})
+@SuppressStaticInitializationFor("org.sunbird.common.ElasticSearchUtil")
+@PowerMockIgnore({"javax.management.*"})
+public class FeedServiceImplTest {
+  private ElasticSearchService esUtil;
+  private static CassandraOperation cassandraOperation = null;
+  private static IFeedService feedService = FeedFactory.getInstance();
+
+  @Before
+  public void setUp() throws Exception {
+    PowerMockito.mockStatic(ServiceFactory.class);
+    PowerMockito.mockStatic(EsClientFactory.class);
+    PowerMockito.mockStatic(ElasticSearchHelper.class);
+    PowerMockito.mockStatic(FeedServiceImpl.class);
+    esUtil = mock(ElasticSearchService.class);
+    esUtil = mock(ElasticSearchRestHighImpl.class);
+    when(EsClientFactory.getInstance(Mockito.anyString())).thenReturn(esUtil);
+    cassandraOperation = mock(CassandraOperationImpl.class);
+    PowerMockito.when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    when(FeedServiceImpl.getCassandraInstance()).thenReturn(cassandraOperation);
+    when(FeedServiceImpl.getESInstance()).thenReturn(esUtil);
+  }
+
+  private static void initCassandraForSuccess() {
+    Response response = new Response();
+    Map<String, Object> responseMap = new HashMap<>();
+    responseMap.put(Constants.RESPONSE, Arrays.asList(getFeedMap()));
+    response.getResult().putAll(responseMap);
+    PowerMockito.when(
+            cassandraOperation.getRecordsByProperties(Mockito.any(), Mockito.any(), Mockito.any()))
+        .thenReturn(response);
+
+    Response upsertResponse = new Response();
+    Map<String, Object> responseMap2 = new HashMap<>();
+    responseMap2.put(Constants.RESPONSE, Constants.SUCCESS);
+    upsertResponse.getResult().putAll(responseMap2);
+    PowerMockito.when(cassandraOperation.upsertRecord(Mockito.any(), Mockito.any(), Mockito.any()))
+        .thenReturn(upsertResponse);
+    PowerMockito.when(
+            cassandraOperation.deleteRecord(
+                Mockito.anyString(), Mockito.anyString(), Mockito.anyString()))
+        .thenReturn(upsertResponse);
+  }
+
+  @Test
+  public void testInsert() {
+    initCassandraForSuccess();
+    Response res = feedService.insert(getFeed(false));
+    Assert.assertTrue(
+        ((String) res.getResult().get(JsonKey.RESPONSE)).equalsIgnoreCase(JsonKey.SUCCESS));
+  }
+
+  @Test
+  public void testUpdate() {
+    initCassandraForSuccess();
+    Response res = feedService.update(getFeed(true));
+    Assert.assertTrue(
+        ((String) res.getResult().get(JsonKey.RESPONSE)).equalsIgnoreCase(JsonKey.SUCCESS));
+  }
+
+  @Test
+  public void testDelete() {
+    initCassandraForSuccess();
+    try {
+      feedService.delete("123-456-789");
```

Comment:
delete method is not throwing any exception ,so code inside exception won't execute.
boolean response = fasle;
 feedService.delete("123-456-789");
response = true;
Assert.assertTrue(response);